### PR TITLE
New competition mail text update

### PR DESCRIPTION
**Pull request by Fernando-Saez**

### CODE REVIEW

File: app/views/notification_mailer/notify_of_new_competition.html.erb [hunk 1]
```diff
@@ -1,13 +1,16 @@
 Queridos socios,
 
 <p>
-  La competición <%= @competition.name %> se anunció ayer. Como de costumbre, encontraréis toda la información al respecto en <%= link_to "la página de la WCA", @competition.website %>.
+  Recientemente se ha aprobado un nuevo campeonato oficial: <%= @competition.name %>. En <%= link_to "la página de la WCA", @competition.website %> encontrarás toda la información importante para registrarte cuanto antes. Si tienes alguna duda, desde la página podrás ponerte en contacto con los organizadores por correo.
```
Comment (AlbertoPdRF) [inline app/views/notification_mailer/notify_of_new_competition.html.erb]:
```suggestion
  Recientemente se ha aprobado una nueva competición oficial: <%= @competition.name %>. En <%= link_to "la página de la WCA", @competition.website %> encontrarás toda la información importante para registrarte cuanto antes. Si tienes alguna duda, desde la página podrás ponerte en contacto con los organizadores por correo.
```
"campeonato" != "competición"

File: app/views/notification_mailer/notify_of_new_competition.html.erb [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
-  Estáis recibiendo esta notificación porque así lo seleccionasteis en vuestro perfil. Para dejar de recibir notificaciones como esta, podéis desmarcar la casilla en <%= link_to "vuestro perfil", profile_url %>.
+  ¡Esperamos verte por allí!
+</p>
+
+<p>
+Estáis recibiendo esta notificación porque así lo seleccionasteis en vuestro perfil. Para dejar de recibir notificaciones como esta, podéis desmarcar la casilla en <%= link_to "vuestro perfil", profile_url %>.
 </p>
 
-Atentamente,
 <br />
```
Comment (AlbertoPdRF) [inline app/views/notification_mailer/notify_of_new_competition.html.erb]:
```suggestion
```

File: app/views/notification_mailer/notify_of_new_competition.text.erb [hunk 3]
```diff
@@ -1,8 +1,9 @@
 Queridos socios,
 
-La competición <%= @competition.name %> se anunció ayer. Como de costumbre, encontraréis toda la información al respecto en <%= link_to "la página de la WCA", @competition.website %>.
+Recientemente se ha aprobado un nuevo campeonato oficial: <%= @competition.name %>. En <%= link_to "la página de la WCA", @competition.website %> encontrarás toda la información importante para registrarte cuanto antes. Si tienes alguna duda, desde la página podrás ponerte en contacto con los organizadores por correo.
```
Comment (AlbertoPdRF) [inline app/views/notification_mailer/notify_of_new_competition.text.erb]:
```suggestion
Recientemente se ha aprobado una nueva competición oficial: <%= @competition.name %>. En <%= link_to "la página de la WCA", @competition.website %> encontrarás toda la información importante para registrarte cuanto antes. Si tienes alguna duda, desde la página podrás ponerte en contacto con los organizadores por correo.
```

File: app/views/notification_mailer/notify_of_new_competition.html.erb [hunk 4]
```diff
@@ -1,13 +1,16 @@
...
+  Recientemente se ha aprobado una nueva campetición oficial: <%= @competition.name %>. En <%= link_to "la página de la WCA", @competition.website %> encontraréis toda la información importante para registrarse cuanto antes. Si tenéis alguna duda, desde la página podréis poneros en contacto con los organizadores por correo.
 </p>
 
 <p>
-  Estáis recibiendo esta notificación porque así lo seleccionasteis en vuestro perfil. Para dejar de recibir notificaciones como esta, podéis desmarcar la casilla en <%= link_to "vuestro perfil", profile_url %>.
+  ¡Esperamos veros por allí!
+</p>
+
+<p>
+Estáis recibiendo esta notificación porque así lo seleccionasteis en vuestro perfil. Para dejar de recibir notificaciones como esta, podéis desmarcar la casilla en <%= link_to "vuestro perfil", profile_url %>.
```
Comment (AlbertoPdRF) [inline app/views/notification_mailer/notify_of_new_competition.html.erb]:
```suggestion
  Estáis recibiendo esta notificación porque así lo seleccionasteis en vuestro perfil. Para dejar de recibir notificaciones como esta, podéis desmarcar la casilla en <%= link_to "vuestro perfil", profile_url %>.
```
Correct indentation.

File: app/views/notification_mailer/notify_of_new_competition.html.erb [hunk 5]
```diff
@@ -1,13 +1,15 @@
 Queridos socios,
 
 <p>
-  La competición <%= @competition.name %> se anunció ayer. Como de costumbre, encontraréis toda la información al respecto en <%= link_to "la página de la WCA", @competition.website %>.
+  Recientemente se ha aprobado una nueva campetición oficial: <%= @competition.name %>. En <%= link_to "la página de la WCA", @competition.website %> encontraréis toda la información importante para registrarse cuanto antes. Si tenéis alguna duda, desde la página podréis poneros en contacto con los organizadores por correo.
```
Comment (Ivan-Brigidano) [inline app/views/notification_mailer/notify_of_new_competition.html.erb]:
```suggestion
  Recientemente se ha aprobado una nueva competición oficial: <%= @competition.name %>. En <%= link_to "la página de la WCA", @competition.website %> encontraréis toda la información importante para registrarse cuanto antes. Si tenéis alguna duda, desde la página podréis poneros en contacto con los organizadores por correo.
```

**Pull request merged**